package Day28_Exceptions;

public class Topic9_GB_Finally {
    public static void main(String[] args) {
        try {
            System.out.println("A code can give exc exception");
            int a = 50 / 0;
        } catch () {
    }
}
